src/itest/kotlin/nl/info/zac/itest/EnkelvoudigInformatieObjectRestServiceTest.kt

```diff
@@ -61,14 +62,14 @@ class EnkelvoudigInformatieObjectRestServiceTest : BehaviorSpec({
     ) {
         When(
             """
-                the create enkelvoudig informatie object with file upload endpoint is called for the zaak with a PDF file
-                """
+            the create enkelvoudig informatie object with file upload endpoint is called for the zaak with a PDF file
+            """
         ) {
             val endpointUrl =
                 "$ZAC_API_URI/informatieobjecten/informatieobject/$zaakProductaanvraag1Uuid/$zaakProductaanvraag1Uuid"
             logger.info { "Calling $endpointUrl endpoint" }
             val file = Thread.currentThread().contextClassLoader.getResource(TEST_PDF_FILE_NAME).let {
```

Comment:
```suggestion
            val file = Thread.currentThread().contextClassLoader.getResource(TEST_PDF_FILE_NAME)?.let {
```

Then the `!!` is no longer necessary